### PR TITLE
Don't allow scaling of in progress k8s deployments

### PR DESCRIPTION
**Pull request by spadgett**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1383042

@jwforres PTAL

### CODE REVIEW

File: app/scripts/controllers/replicaSet.js [hunk 1]
```diff
@@ -280,6 +281,33 @@ angular.module('openshiftConsole')
...
+              resource: 'replicasets'
+            }, context, function(replicaSets) {
+              var deploymentSelector = new LabelSelector($scope.deployment.spec.selector);
+              inProgressDeployment = false;
+
+              // See if there is more than one replica set that matches the
+              // deployment selector with active replicas.
+              var numActive = 0;
+              _.each(replicaSets.by('metadata.name'), function(replicaSet) {
+                if (!deploymentSelector.matches(replicaSet) || !replicaSet.status.replicas) {
```
Comment (jwforres) [inline app/scripts/controllers/replicaSet.js]: is matches the right check here?  shouldn't it be comparing the labelSelector of the deployment to the labelSelector of the replicaSet?  matches is going to checks the labels on the replicaSet, covers would check the labelSelector

**Comment by spadgett** [comment]
@jwforres thanks, updated

**Comment by jwforres** [comment]
@spadgett were you using matches anywhere else in the deployment/replicaset rollup that we should have been using covers instead?

**Comment by jwforres** [comment]
actually i see you updated another one in your latest commit, thanks  [merge]

**Pull request merged**